src/main/java/org/springframework/data/couchbase/core/query/QueryCriteria.java

```diff
@@ -52,77 +66,177 @@ public QueryCriteria or(String key) {
 		return new QueryCriteria(this.criteriaChain, key, null, ChainOperator.OR);
 	}
 
+	public QueryCriteria child(QueryCriteria criteria) {
```

Comment:
does the `child` naming align with similar implementations in other spring-data projects?

Reply:
Agree - child is not a good name.  I've changed it to 'wrap' and also made it private.